CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt

```diff
@@ -1,6 +1,9 @@
 == 3030-12-31 Getting Started
 <p>If you're new to Constellation, read the <a href="" helpId="au.gov.asd.tac.constellation.functionality.gettingstarted">getting started guide</a>.</p>
 
+== 2022-03-16 Data Access View
+<p>A new tab has been added in SetUp -> Options -> Data Access View to hide data access categories that are not used by the users.</p>
```

Comment:
I think I'd change the last part from "to hide data access categories that are not used by the users" to "to be able to hide data access categories that you don't want to see". Makes the entry sound more directed at users (which I think it should be).